assets/js/base/hocs/with-reviews.js

```diff
@@ -88,30 +101,10 @@ const withReviews = ( OriginalComponent ) => {
 				return;
 			}
 
-			const args = {
-				...this.getArgs(),
-				per_page: reviewsToDisplay - reviews.length,
-				offset: reviews.length,
-			};
-
-			this.updateListOfReviews( args, reviews ).then( ( { newReviews } ) => {
-				if ( announceUpdates ) {
-					speak(
-						sprintf(
-							_n(
-								'%d review loaded.',
-								'%d reviews loaded.',
-								newReviews.length,
-								'woo-gutenberg-products-block'
-							),
-							newReviews.length,
-						)
-					);
-				}
-			} );
+			this.updateListOfReviews( reviews ).then( onReviewsAppended );
 		}
 
-		updateListOfReviews( args, oldReviews = [] ) {
+		updateListOfReviews( oldReviews = [] ) {
```

Comment:
Ya I see why you have this now, because replace reviews passes in no arg hence forcing a reset of the reviews listed 👍